### PR TITLE
introducing a Zeldovich sim data source 

### PR DESCRIPTION
**Pull request by nickhand**

### CODE REVIEW

File: nbodykit/plugins/datasource/ZeldovichSim.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+from nbodykit.extensionpoints import DataSource
+from nbodykit import utils
+
+import numpy
+import logging
+
+logger = logging.getLogger('ZeldovichSim')
+         
+class ZeldovichSimDataSource(DataSource):
```
Comment (rainwoodman) [inline nbodykit/plugins/datasource/ZeldovichSim.py]: I wonder if it is the time to add a FastPM python binding such that we can use some of the routines there, for 'sim' datasets.  e.g. the routine to fill the linear gaussian field with gadget random seeds could be useful here. Then we can also avoid reading the simulation from disk etc.

**Comment by rainwoodman** [comment]
I am guessing conda comes with a ready-to-use fortran compiler that may work better than the ubuntu one. Does class require fortran?

**Comment by nickhand** [comment]
victory! 

A lot of the issues were coming from the `classylss` side of things. basically, we just need to specify CXX and F90 to their GNU compiler versions, i.e., g++ and gfortran.

The fortran dependency comes from including the source code for FFTLog, which is used for power spectrum <--> correlation functions

File: nbodykit/plugins/datasource/ZeldovichSim.py (hunk 1 above)
Comment (nickhand) [inline nbodykit/plugins/datasource/ZeldovichSim.py]: that's an interesting idea...we'll certainly need to start increasing the synergy between FastPM and nbodykit once we get going on the emulator a bit more, so this is definitely worth thinking about. How much work would it be?

Comment (rainwoodman) [inline nbodykit/plugins/datasource/ZeldovichSim.py]: I crafted a 'proper' C++ binding in around 16 hours. It wouldn't be too much more difficult to do something like this with Cython.

**Comment by rainwoodman** [comment]
we need "matching" cmpilers with classlss with class and fft log?

Does this mean we need class and fftlog installed before pip install classlss?

**Comment by nickhand** [comment]
The CLASS compilation is independent -- `classylss` downloads CLASS and uses the built-in Makefile. The default use `gcc` to compile the `.c` files of CLASS to object files. 

Then `classylss`  uses the `gfortran` to compile the `.f` files of fftlog to object files, and uses the C++ compiler `g++` to compile the main library of `classylss`. Then all of the object files are linked together into a library.

I'm by no means an expert here, so let me know if we should be doing something else

**Comment by rainwoodman** [comment]
I see. I forgot classlss is CPP!

**Comment by nickhand** [comment]
okay I am going to merge this and close #197

**Pull request merged**